virt_lightning/shell.py

```diff
@@ -121,14 +124,26 @@ def ansible_inventory(configuration, context, **kwargs):
         '-o StrictHostKeyChecking=no"'
     )
 
+    groups = {}
     for domain in hv.list_domains():
+        for group in domain.groups:
+            if group not in groups:
```

Comment:
I think more better use here:

```python
if not groups.get(group, None):
    groups[group] = []
groups[group].append(domain)
```

Reply:
I'm not convinced Veniamin. I believe it's a good use-case for a `in`.

Reply:
As you wish :-D